loops-and-arrays.js

```diff
@@ -3,8 +3,15 @@
 // Schrijf een script dat iedere naam in bovenstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
+console.log("------opdracht 1------")
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+console.log(names);
+
+for (let newNames = 0; newNames < names.length; newNames++) {
```

Comment:
Je mag als variabelenaam bij de for-loops ook gewoon i laten staan, ik denk dat dat de leesbaarheid / herkenbaarheid wat ten goede komt!